src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -228,6 +234,86 @@ internal static WorkerInitResponse WorkerInitRequestHandler(WorkerInitRequest re
             return response;
         }
 
+        internal static async Task<FunctionMetadataResponses> FunctionsMetadataRequestHandler(FunctionsMetadataRequest request)
```

Comment:
Can we rename the method as FunctionsMetadataRequestHandler**Async**? Generally we do this for Async methods.

Reply:
yes good catch!